doc/lib.rst

```diff
@@ -595,9 +573,21 @@ Network Programming and Internet Protocols
   Wrapper for OpenSSL.
 
 
-
 Unstable
 ========
 
 * `atomics <atomics.html>`_
   Types and operations for atomic operations and lockless algorithms.
+
```

Comment:
>  I didn't move them to the "Unstable" section, because I think they fit better in their current category.

indeed, a unstable section is the wrong way to organize things; also, some modules have stable and unstable APIs; for modules that are indeed unstable/experimental, we could just defer to the docs on those modules

Reply:
Makes sense, I removed the "Unstable" category.